bootstrap/scripts/buildPatch.sh

```diff
@@ -1,10 +1,18 @@
-if [ $PUBLISH_ENDPOINT = "local" ]; then
+if [ $PUBLISH_ENDPOINT = "local" ] && [ -z $FAST ] ; then
 	echo "start verdaccio"
 	verdaccio > verdaccio.log 2>&1 &
 	FOO_PID=$!
 	echo "$FOO_PID"
 	sleep 3
 fi
+if [ $PUBLISH_ENDPOINT = "local" ] && [ $FAST ] ; then
+	echo "start verdaccio in fast mode"
```

Comment:
instead of calling it 'fast mode', can we describe it for what's actually happening? i.e. `starting verdaccio with in-memory cache to speed up build time`

Reply:
updated